server/graphql_schemas/views.py

```diff
@@ -3,16 +3,14 @@
 from graphene_django.views import GraphQLView
 from rest_framework.decorators import authentication_classes, permission_classes, api_view
 from rest_framework.settings import api_settings
+from graphene_file_upload.django import FileUploadGraphQLView
+
 from graphql_schemas.utils.helpers import UnauthorizedCalendarError
 
 from graphql.error.located_error import GraphQLLocatedError
 
 
-class DRFAuthenticatedGraphQLView(GraphQLView):
-    def parse_body(self, request):
-        if isinstance(request, rest_framework.request.Request):
```

Comment:
@j33n Any reason why this function is been removed?

Reply:
Hey @makindemayowa, I removed it since the same thing was being done in the `FileUploadGraphQLView` function and I was inheriting from it. Also if both functions are there an error is thrown